static/js/containers/ProfileFormContainer.js

```diff
@@ -98,7 +98,13 @@ class ProfileFormContainer extends React.Component {
         dispatch(clearProfileEdit(username));
       });
     } else {
-      return Promise.reject(errors);
+      return Promise.reject(errors).catch(() => {
```

Comment:
I tried that but I couldn't get it to work, I think there's a race condition between dispatching the validation errors above and react re-rendering the page, so if we look for the `.invalid-input` selector on the page it hasn't shown up yet. Doing this (putting the `scrollIntoView` code in the `.catch` block) is the only way I found to ensure that the code to find the selector executed after the DOM had actually been updated.

Do you have any ideas for how to organize it better?